### PR TITLE
persona redesign

### PR DESCRIPTION
**Pull request by Tlaster**

### CODE REVIEW

File: common/src/androidMain/kotlin/com/dimension/maskbook/common/ui/widget/button/MaskListButton.kt [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import com.dimension.maskbook.common.ext.applyTextStyle
+import com.dimension.maskbook.common.ui.widget.MaskListItem
+
+@Composable
+fun MaskListButton(
```
Comment (qdsfdhvh) [inline common/src/androidMain/kotlin/com/dimension/maskbook/common/ui/widget/button/MaskListButton.kt]: Maybe rename to `MaskListItemButton`?

File: persona/src/androidMain/kotlin/com/dimension/maskbook/persona/ui/scenes/avatar/SetAvatarScene.kt [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    val currentAvatar by viewModel.currentAvatar.collectAsState(null)
+    MaskScene {
+        MaskScaffold(
+            topBar = {
+                MaskSingleLineTopAppBar(
+                    navigationIcon = {
+                        MaskBackButton(onBack = onBack)
+                    },
+                    title = {
+                        Text(text = "Set Persona Avatar")
```
Comment (JoeSteven) [inline persona/src/androidMain/kotlin/com/dimension/maskbook/persona/ui/scenes/avatar/SetAvatarScene.kt]: put hardcode strings into app.json

**Review by JoeSteven — APPROVED** [review]
LGTM!

**Review by qdsfdhvh — APPROVED** [review]
LGTM!

**Review by HuixingWong — APPROVED** [review]
LGTM!

**Pull request merged**